ntbs-service/Startup.cs

```diff
@@ -46,6 +49,20 @@ public void ConfigureServices(IServiceCollection services)
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
+            services.AddHangfire(config => config
+                .SetDataCompatibilityLevel(CompatibilityLevel.Version_170)
+                .UseSimpleAssemblyNameTypeSerializer()
+                .UseRecommendedSerializerSettings()
+                .UseSqlServerStorage(@"data source=.\NTBSSERVER;initial catalog=ntbsHangfire;trusted_connection=true", new SqlServerStorageOptions
```

Comment:
Does the connection string want to be parameter-ised out? Also if in dev we require adding an instance of name `NTBSSERVER` this probably wants to be added to a readme

Reply:
I think that's just Farman's local db. Yes, we definitely want to parametric this, probably just following the same pattern as other for the main db!